capella_model_explorer/__main__.py

```diff
@@ -0,0 +1,362 @@
+# Copyright DB InfraGO AG and contributors
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import os
+import pathlib
+import subprocess
+import threading
+import time
+
+import capella_model_explorer
+import capella_model_explorer.constants as c
+
+
+def _check_cmd_run(cmd: list[str], errmsg: str = "") -> None:
+    """Check if a command is available."""
+    try:
+        subprocess.run(
+            cmd,
+            check=True,
+            capture_output=True,
+        )
+    except subprocess.CalledProcessError:
+        print(f"Command '{cmd[0]}' not found. {errmsg}")
+        raise SystemExit(1) from None
```

Comment:
This error message is very misleading, an arbitrary command can fail for any number of reasons. `npm install` for instance can fail if the network is unavailable, even though npm itself is installed.